package pyrock.events.window;


import rockstar.client.event.*;
import lombok.Generated;
import rockstar.client.event.ClientEvent;

public class ChatScrollEvent
extends ClientEvent {
    private final float x;
    private final float y;
    private final float horizontal;
    private final float vertical;

    @Generated
    public float getX() {
        return this.x;
    }

    @Generated
    public float getY() {
        return this.y;
    }

    @Generated
    public float getHorizontal() {
        return this.horizontal;
    }

    @Generated
    public float getVertical() {
        return this.vertical;
    }

    @Generated
    public ChatScrollEvent(float f, float f2, float f3, float f4) {
        this.x = f;
        this.y = f2;
        this.horizontal = f3;
        this.vertical = f4;
    }
}

